packages/odata-service-inquirer/test/unit/index-api.test.ts

```diff
@@ -128,10 +125,6 @@ describe('API tests', () => {
                 "validate": [Function],
                 "when": [Function],
               },
-              {
-                "name": "capCliStateSetter",
```

Comment:
For this test, `getPrompts` is called with param `isYUI` set to true. So a CLI prompt should not have been returned